sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -21,8 +21,11 @@
 
 import org.sonar.api.Plugin;
 import org.sonar.api.PropertyType;
+import org.sonar.api.SonarProduct;
+import org.sonar.api.SonarQubeVersion;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.api.SonarQubeVersion'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-235b&issues=AWDqoxoRo_TTbi26M253&open=AWDqoxoRo_TTbi26M253)

[ISSUE_KEY]: AWDqoxoRo_TTbi26M253